src/blueprints/resources.py

```diff
@@ -430,12 +430,25 @@ def user_not_authorized(response):
 
     def get_default_ownership_policy_cfg(resource_id, uid, action):    
         return { "resource_id": resource_id, "action": action, "rules": [{ "AND": [ {"EQUAL": {"id" : uid } }] }] }
+    
+    def get_default_resource_policy_cfg(resource_id, action):    
+        return { "resource_id": resource_id, "action": action, "rules": [{ "AND": [ {"EQUAL": {"isOperator" : True } }] }] }
```

Comment:
Prefer to merge both methods together, such as:
def get_default_resource_policy_cfg(resource_id, action):
    if uid:
        return { "resource_id": resource_id, "action": action, "rules": [{ "AND": [ {"EQUAL": {"id" : uid } }] }] }
    return { "resource_id": resource_id, "action": action, "rules": [{ "AND": [ {"EQUAL": {"isOperator" : True } }] }] }
    
This is to avoid extra methods that make the code harder to maintain in the future.